feat(docs): add landing page (#215)
rocket-connect/graphql-debugger: docs/src/pages/index.mdx

> ---
> title: GrapqhQL Debugger
> searchable: false
> ---

Reviewer: ```suggestion
```

Lets remove this for the time being as we need to do a round of SEO on all pages.

{/* wrapped with {} to mark it as javascript so mdx will not put it under a p tag */}
{<h1 className="text-center font-extrabold md:text-5xl mt-8">GraphQL Debugger</h1>}

{/* Icons go here: Typescript, OTEL, MIT, Graphql */}

> GraphQL Debugger is a OpenTelemetry collector with a user interface tailored for debugging GraphQL servers.
> It is designed to be used during development to help you identify errors, slow queries, and inconsistencies without having to rely on outdated methods like logging.

Reviewer: ```suggestion
GraphQL Debugger is a platform for debugging your GraphQL Server, it helps you identify errors, slow queries, and inconsistencies without having to rely on outdated methods like logging.
```
